code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -385,15 +385,26 @@
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/hundred
 
 /obj/item/gun/ballistic/shotgun/winchester
-	name = "Winchester 2073"
-	desc = "A sturdy lever action rifle. Although this model is a antique model, the design goes even further back, and has been proven reliable over 600 years."
+	name = "Winchester MK.2"
+	desc = "A sturdy lever action rifle. This one is a newer reproduction."
+	icon_state = "winchester"
+	item_state = "winchester"
 	icon = 'icons/obj/guns/48x32guns.dmi'
 	mob_overlay_icon = 'icons/mob/clothing/back.dmi'
 	lefthand_file = 'icons/mob/inhands/weapons/64x_guns_left.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/64x_guns_right.dmi'
-	icon_state = "winchester"
-	item_state = "winchester"
 	inhand_x_dimension = 32
 	inhand_y_dimension = 32
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/winchester
 	fire_sound = 'sound/weapons/gun/rifle/shot.ogg'
+
+/obj/item/gun/ballistic/shotgun/winchester/lethal
+	mag_type = /obj/item/ammo_box/magazine/internal/shot/winchester/lethal
+
+/obj/item/gun/ballistic/shotgun/winchester/mk1
+	name = "Winchester MK.1"
+	desc = "A sturdy lever action rifle. This one is an antique and old model."
```

Comment:
desc = "A sturdy lever action rifle. This older pattern appears to be an antique, in excellent condition despite its age."